sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -15,7 +15,7 @@ public static LusidInstrument GetExampleInstrument(string instrumentName)
                 nameof(Bond) => CreateExampleBond(),
                 nameof(FxForward) => CreateExampleFxForward(),
                 nameof(FxOption) => CreateExampleFxOption(),
-                nameof(InterestRateSwap) => CreateExampleInterestRateSwap(),
+                nameof(InterestRateSwap) => CreateExampleInterestRateSwap("Vanilla"),
```

Comment:
what are the other valid options? can we add a comment pointing to where they can find the list (no need to list inline)

Reply:
Yes will do.